docs/book/advanced-guide/getting_started_on_gcp.md

```diff
@@ -0,0 +1,293 @@
+pipelines orchestrator. This is a serverless GCP specific offering with minimal
+hassle.
+* A **container registry** for pushing and pulling the pipeline image.
+* Finally, the **secrets manager** to store passwords and ssl certificates.
+
+## Step 1/10 Set up a GCP project (Optional)
+
+As a first step it might make sense to 
+[create](https://console.cloud.google.com/projectcreate) 
+a separate GCP project for your zenml resources. However, this step is 
+completely optional, and you can also move forward within an existing project. 
+If some resources already exist, feel free to skip the creation of the 
+resources and simply note down the relevant information. 
+
+For simplicity, just
+open up a terminal on the side and export relevant values as we go along. You
+will use these when we set up the ZenML stack.
+ZenML will use your project number at a later stage to connect to some 
+resources, so let's export it. You'll most probably find it right 
+[here](https://console.cloud.google.com/welcome).
+
+```bash
+export PROJECT_NUMBER=<PROJECT_NUMBER> # for example '492014921912'
+```
+
+## Step 2/10 Enable billing
+
+Before moving on, you'll have to make sure you attach a billing account to 
+your project. In case you do not have the permissions to do so, you'll have to
+ask an organization administrator to do so. 
+[Here](https://console.cloud.google.com/billing/projects) is the relevant 
+page to do so.
+
+## Step 3/10 Enable Vertex AI
+
+Vertex AI pipelines is at the heart of our GCP stack. As the orchestrator 
+Vertex AI will run your pipelines and use all the other stack components. 
+All you'll need to do at this stage is enable Vertex AI
+[here](https://console.cloud.google.com/vertex-ai).
+
+Once it is enabled you will see a drop-down with all the regions where 
+Vertex AI is available. At this point it might make sense to make this 
+decision for your ZenML Stack and export the full region name.
+
+```shell
+export GCP_LOCATION=<GCP_LOCATION> # for example 'europe-west3'
+```
+
+## Step 4/10 Enable Secrets Manager
+
+The secrets manager will be needed so that the orchestrator will have secure
+access to the other resources. 
+[Here](https://console.cloud.google.com/marketplace/product/google/secretmanager.googleapis.com)
+is where you'll be able to enable the secrets manager.
+
+## Step 5/10 Enable Container Registry
+
+The Vertex AI orchestrator uses Docker Images containing your pipeline code
+for pipeline orchestration. For this to work you'll need to enable the GCP
+docker registry 
+[here](https://console.cloud.google.com/marketplace/product/google/containerregistry.googleapis.com).
+
+In order to use the container registry at a later point you will need to 
+set the container registry uri. This is how it is usually constructed:
+`gcr.io/<PROJECT_ID>`. 
+
+
+{% hint style="info" %}
+The container registry has four options: **gcr**.io , us.gcr.io , eu.gcr.io , 
```

Comment:
Does it look better if we format this into `gcr.io` , `us.gcr.io` , `eu.gcr.io`?